src/main/java/com/google/firebase/auth/FirebaseTokenUtils.java

```diff
@@ -87,8 +87,9 @@ static FirebaseTokenVerifierImpl createIdTokenVerifier(
         clock, ID_TOKEN_ISSUER_PREFIX, projectId);
     GooglePublicKeysManager publicKeysManager = newPublicKeysManager(
         app.getOptions(), clock, ID_TOKEN_CERT_URL);
+    String shortName = Utils.isEmulatorMode() ? "Emulator ID token" : "ID token";
```

Comment:
I don't think this is necessary. Let's use the same error messages/text as the production mode. That's how this is implemented in other languages too.

Reply:
Done.